sphinxcontrib/datatemplates/directive.py

```diff
@@ -1,67 +1,30 @@
 import json
-import mimetypes
-import defusedxml.ElementTree as ET
+import csv
 
+import defusedxml.ElementTree as ET
+import yaml
 from docutils import nodes
 from docutils.parsers import rst
 from docutils.statemachine import ViewList
 from sphinx.util import logging
 from sphinx.util.nodes import nested_parse_with_titles
-import csv
-import yaml
 
 from sphinxcontrib.datatemplates import helpers
 
 LOG = logging.getLogger(__name__)
 
 
-def _handle_dialect_option(argument):
-    return rst.directives.choice(argument, ["auto"] + csv.list_dialects())
-
-
-class DataTemplate(rst.Directive):
+class DataTemplateBase(rst.Directive):
 
     option_spec = {
-        'source': rst.directives.unchanged,
-        'template': rst.directives.unchanged,
-        'csvheaders': rst.directives.flag,
-        'csvdialect': _handle_dialect_option,
+        'source': rst.directives.unchanged_required,
+        'template': rst.directives.unchanged_required,
     }
     has_content = False
 
     def _load_data(self, env, data_source):
         rel_filename, filename = env.relfn2path(data_source)
```

Comment:
If we aren't going to use the file, we don't need to search for it, so this line could come out, too.

Reply:
I don't actually understand what you mean. Just remove line 26?